src/main/java/teammates/common/util/SanitizationHelper.java

```diff
@@ -360,4 +360,25 @@ public static String sanitizeStringForXPath(String text) {
         }
         return "concat(" + result.toString() + "'')";
     }
+
+    /**
+     * Returns true if the {@code string} has evidence of having been sanitized.
+     * A string is considered sanitized if it does not contain any of the chars '<', '>', '/', '\"', '\'',
+     * and contains at least one of their sanitized equivalents or the sanitized equivalent of '&'.
+     */
+    public static boolean isSanitizedHtml(String string) {
+        return string != null
+                && !StringHelper.isTextContainingAny(string, "<", ">", "\"", "/", "\'")
```

Comment:
The reason this check is here is because "<p> &gt; </p>" should not be considered sanitized, while "&gt;" is.